### ISSUE TITLE
There is a bug

### ISSUE THREAD

**Issue by ross-Hr** [taimurhassan/tensorpooling#2]
when i run the code ,in predict.py，the bug come from 
from models.all_models import model_from_name.

there is no file called models.all_models.

Are you referring to the repositoriy of  https://github.com/divamgupta/image-segmentation-keras

**Comment by taimurhassan** [comment]
Yes, this repo is derived from divamgupta's work. But models.all_models is not required here. Let me know which error are you getting at your end.

**Comment by ross-Hr** [comment]
yes,you're right，The module of all_models.py needs to be referenced inside the function
